### ISSUE TITLE
[BUG] Missing phrase

### ISSUE THREAD

**Issue by Troughy** [qbcore-framework/qb-core#1010]
### Summary

info.checking_ban and info.join_server are not in locale files, but they are used in the code, resulting in "Missing phrase for key" warnings

### Reproduction

just join the server

### Expected behavior

no warnings

### Actual behavior

someone forgot to put these into locale files so we get these warnings
Warning: Missing phrase for key: "info.checking_ban"
Warning: Missing phrase for key: "info.join_server"

### Last Updated

newest version

### Custom Resources

doesn't matter

### Resource Rename

no

**Comment by Edvo1901** [comment]
Yes it does. At least for English, it does. You should specific which language tho

**Comment by Troughy** [comment]
> Yes it does. At least for English, it does. You should specific which language tho

You can see the code for yourself, check with GitHub.dev (press .(dot)) and search for the keys, you will see they're referenced somewhere but not in any locale files

**Comment by Edvo1901** [comment]
It does, i just check 4 locales file. It all does, perhaps you could just tell me what your locale is?

**Comment by Troughy** [comment]
Just got home, idk what drugs I was on yesterday when I checked it, I probably searched for `info.checking_ban` and that's why I didn't find it... We're using the hu.lua locale and it is missing there, so im just gonna leave this issue open

**Comment by GitMocha** [comment]
You can always PR a hu locale edit with the missing translations. Alternatively copy a locale that has those in and just edit to the language you're using for yourself.

> Just got home, idk what drugs I was on yesterday when I checked it, I probably searched for `info.checking_ban` and that's why I didn't find it... We're using the hu.lua locale and it is missing there, so im just gonna leave this issue open

**Comment by Maxhero54** [comment]
> ### Summary
> info.checking_ban and info.join_server are not in locale files, but they are used in the code, resulting in "Missing phrase for key" warnings
> 
> ### Reproduction
> just join the server
> 
> ### Expected behavior
> no warnings
> 
> ### Actual behavior
> someone forgot to put these into locale files so we get these warnings Warning: Missing phrase for key: "info.checking_ban" Warning: Missing phrase for key: "info.join_server"
> 
> ### Additional context
> _No response_
> 
> ### Last Updated
> newest version
> 
> ### Custom Resources
> doesn't matter
> 
> ### Resource Rename
> no

**Solution** :   server.cfg         
 
# QBCore locale config
setr qb_locale "en"
